utils/python/CIME/system_test.py

```diff
@@ -538,14 +573,16 @@ def _consumer(self, test, test_phase, phase_method):
         before_time = time.time()
         success = self._run_catch_exceptions(test, test_phase, phase_method)
         elapsed_time = time.time() - before_time
-        status  = (TEST_PENDING_STATUS if test_phase == RUN_PHASE and not self._no_batch else TEST_PASS_STATUS) if success else TEST_FAIL_STATUS
+        status  = (TEST_PENDING_STATUS if test_phase == RUN_PHASE and not \
+                   self._no_batch else TEST_PASS_STATUS) if success else TEST_FAIL_STATUS
 
-        if (status != TEST_PENDING_STATUS):
+        if status != TEST_PENDING_STATUS:
             self._update_test_status(test, test_phase, status)
         self._handle_test_status_file(test, test_phase, success)
 
```

Comment:
I see you got frustrated with all the long lines. What number of characters would you like to stay under? 80 is common but I'd prefer 100.